### PR TITLE
Create skeleton

### PR DESCRIPTION
**Pull request by rudigiesler**

### CODE REVIEW

**Comment by rudigiesler** [comment]
Ready for review.

File: test/test_xform.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+        app.states.add('states:end', function(name) {
+            return new EndState(name, {
+                text: 'This is the end state.'
+            });
+        });
+
+        tester
+            .setup.config.app({
+                name: 'xformState-tester'
```
Comment (hodgestar) [inline test/test_xform.js]: The casing of `xformState-tester` looked odd to me. Maybe`xform-state-tester`?

**Comment by hodgestar** [comment]
One very stupid bikeshed, otherwise :+1:.

**Comment by rudigiesler** [comment]
Comments addressed, ready for rereview.

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**